Dockerfile

```diff
@@ -79,5 +89,5 @@ RUN pip install .
 #
 
 # set the entry point so that module is called with any parameters given to the `docker run` command
-ENTRYPOINT ["/bin/bash", "/usr/local/icetray/env-shell.sh"]
+#ENTRYPOINT ["/bin/bash", "/usr/local/icetray/env-shell.sh"]
```

Comment:
Why is this no longer needed?

Reply:
As far as I can tell, if the ENTRYPOINT is not defined it defaults to the base image's ENTRYPOINT.  Which is exactly this.  Or at least the tests pass with this commented out?